jekyll/_cci2/webhooks.adoc

```diff
@@ -72,30 +79,134 @@ To configure webhooks within the CircleCI app:
 | You must select at least one event that will trigger a webhook
 |===
 
-^1^Only leave this unchecked for testing purposes.
+=== Inbound webhook setup
+
+Follow these steps to set up and test an inbound webhook trigger for a pipeline from anywhere that can emit a webhook or run a curl command:
+
+{% include snippets/inbound-webhook-setup.adoc %}
+
+. You can now test your inbound webhook trigger with `curl`. To trigger your pipeline, copy and paste the following sample request and replace `<your-URL>` and `<your-secret>` with the URL and secret that you generated in the previous step:
++
+NOTE: When triggering via `curl`, you must use a `POST` request with `content-type: application/json`.
++
+[,shell]
+----
+curl -X POST -H "content-type: application/json" 'https://internal.circleci.com/private/soc/e/<your-URL>?secret=<your-secret>'
+----
+
+See our link:https://discuss.circleci.com/t/trigger-pipelines-from-anywhere-inbound-webhooks-now-in-preview/49864[community forum] for more details or how to use this functionality with a link:https://discuss.circleci.com/t/re-build-automatically-when-new-image-is-available-on-dockerhub/50350[3rd party service like DockerHub].
+
+== Inbound webhooks
+
+Use **inbound webhooks** to trigger a pipeline from anywhere that can emit a webhook or run a curl command. This section presents some use cases for CircleCI inbound webhooks.
+
+=== Trigger one pipeline from another with inbound webhooks
+
+Use an inbound webhook to configure one pipeline to trigger a second pipeline. For example, you might need to run a set of integration tests on a repository after you have made a change to a separate repository.
+
+For this example, assume you have two projects in separate repositories:
```

Comment:
Might be worth adding "but you can do the same for different pipelines within the same project".